queries.sql

```diff
@@ -23,3 +23,95 @@ SELECT * FROM animals WHERE name <> 'Gabumon';
 
 -- Find all animals with a weight between 10.4kg and 17.3kg (including the animals with weights equal to 10.4kg or 17.3kg)
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+
+
+-- Begin the transaction
+BEGIN;
+
+-- Update species for animals whose name ends in "mon"
+UPDATE animals SET species = 'Digimon' WHERE name LIKE '%mon';
+
+-- Update species for animals with no species set
+UPDATE animals SET species = 'Pokemon' WHERE species IS NULL;
+
+-- Verify changes before committing
+SELECT * FROM animals;
+
+-- Commit the transaction
+COMMIT;
```

Comment:
- [ ] It seems like you're missing the transaction for the question "Inside a transaction update the animals table by setting the species column to unspecified. Verify that the change was made. Then roll back the change and verify that the species columns went back to the state before the transaction." which should be before the highlighted SQL transaction.
- [ ] You're screenshot for this transaction is also incomplete so please update it.